fury/ui.py

```diff
@@ -5240,3 +5240,387 @@ def _set_position(self, coords):
+                                   bg_color=self.bullet_bg_color,
+                                   text_color=self.text_color)
+
+        child_node.indent = root_node.indent + 30
+        self.get_last_child(root_node)
+        last_idx = self.nodes.index(self._last_child)
+
+        # Add the new node in dictionary and list
+        root_node.add_node(child_node)
+        self.dict[child_node.node] = child_node
+        self.nodes.insert(last_idx + 1, child_node)
+
+        self.panel.add_element(child_node, (child_node.indent,
+                                            self._last_child.position[1] - 50))
+        self.update_node(root_node)
+
+    def update_node(self, node):
+        """Update the positions of all nodes below the newly added node
+        """
+        new_node = node.children[-1]
+        _idx = self.nodes.index(new_node) + 1
+        _offsetY = 50
+        for node in self.nodes[_idx:]:
+            self.panel.update_element(node, (node.indent,
+                                             new_node.position[1] - _offsetY))
+
+            _offsetY += 50
+
+    def get_last_child(self, node):
+        """Get the previous node to newly added node
+        """
+        if len(node.children) > 0:
+            self.get_last_child(node.children[-1])
+        else:
+            self._last_child = node
+
+    @property
+    def color(self):
+        """ Returns the background color of the bullet point list.
+        """
+
+        return self.panel.color
+
+    @color.setter
+    def color(self, color):
+        """ Sets background color of the bullet point list.
+        Parameters
+        ----------
+        color : list of 3 floats.
+        """
+
+        self.panel.color = color
+
+    def left_button_pressed(self, i_ren, _obj, _sub_component):
+        click_pos = np.array(i_ren.event.position)
+        self._click_position = click_pos
+        i_ren.event.abort()
+
+    def left_button_dragged(self, i_ren, _obj, _sub_component):
+        click_position = np.array(i_ren.event.position)
+        change = click_position - self._click_position
+        self.panel.position += change
+        self._click_position = click_position
+        i_ren.force_render()
+
+
+class BulletPoint2D(UI):
+    """Create a Bullet Point from a node
+    Attributes
+    ----------
+    panel: :class: 'Panel2D'
+        Container to hold UI elements.
+    bullet: :class: 'Disk2D'
+        Displays the bullet before the text.
+    bullet_textbox: :class: 'TextBLock2D'
+        Displays the node as text after the bullet.
+    """
+
+    def __init__(self, node="", bullet_radius=5,
+                 indent=5, draggable=False, size=(300, 20),
+                 position=(0, 0), bg_color=(1, 1, 1),
+                 bg_opacity=1, bullet_color=(0, 0, 0),
+                 text_color=(0, 0, 0)):
+        """
+        Parameters
+        ----------
+        node: str
+            Text to display after the bullet
```

Comment:
I think you should rename this to something else or it will be confusing. Something like `bullet_label` or just `label` works.